dotCMS/src/main/java/com/dotmarketing/cms/myaccount/action/AddressAction.java

```diff
@@ -1,9 +1,11 @@
 package com.dotmarketing.cms.myaccount.action;
 
 import com.dotcms.repackage.org.apache.struts.actions.DispatchAction;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.repackage.org.apache.struts.actions.DispatchAction'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)